projects/uitdatabank/docs/entry-api/events/booking-availability.md

```diff
@@ -1,7 +1,98 @@
-Coming soon.
+# Marking an event as sold out
 
-<!-- 
-  @todo 
-  Explain how to mark an event as sold out.
-  Maybe add a screenshot how sold out events are displayed on UiTinVlaanderen (as an example).
--->
+When [creating](./create.md) or (usually) when [updating](./update.md) your event, you can mark it as sold out to indicate to interested attendees that there are no more tickets/seats/reservations available.
+
+Sold out events automatically get an *"(Volzet of uitverkocht)"* label next to their calendar info in online calendars. For example, on UiTinVlaanderen:
```

Comment:
```suggestion
Events that are sold out or fully booked automatically get an *"(Volzet of uitverkocht)"* label next to their calendar info in online calendars. For example, on UiTinVlaanderen:
```

Reply:
"Sold out" (uitverkocht) is used by partners that sell tickets, "fully booked" (volzet) by partners / organizers that have a limited amount of seats / participants for their event